app/views/layouts/apipie/apipie.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Looks the same as https://github.com/Apipie/apipie-rails/blob/master/app/views/layouts/apipie/apipie.html.erb, is it necessary to override it?